### PR TITLE
Upgrade to selenium3

### PR DESCRIPTION
**Pull request by wjrsimpson**

### CODE REVIEW

**Comment by wjrsimpson** [comment]
[Upgrade to selenium 3](https://trello.com/c/L5Iwmsvb/3417-upgrade-to-selenium-3)

**Comment by wjrsimpson** [comment]
[Upgrade to selenium 3](https://trello.com/c/L5Iwmsvb/3417-upgrade-to-selenium-3)

File: TESTAUTO.md [hunk 1]
```diff
@@ -83,14 +85,14 @@ This should setup the test data and then run a test that verifies the data persi
     * Test class: ```org.orcid.integration.blackbox.BlackBoxTestSuite```
     * Test runner: ```JUnit 4```
 
-6. In the Arguments tab, set the following VM arguments
+6. In the Arguments tab, set the following VM arguments (note that you need to insert your `webdriver.gecko.driver` path)
```
Comment (rcpeters) [inline TESTAUTO.md]: @wjrsimpson we should suggest a place for this `~\bin\`?

File: orcid-integration-test/src/test/java/org/orcid/integration/blackbox/web/shibboleth/ShibbolethTest.java [hunk 2]
```diff
@@ -80,6 +80,9 @@ private void createFireFoxDriverWithModifyHeaders() throws IOException {
         capabilities.setBrowserName("firefox");
         capabilities.setPlatform(org.openqa.selenium.Platform.ANY);
         capabilities.setCapability(FirefoxDriver.PROFILE, fireFoxProfile);
+        // Marionette does not allow untrusted certs yet
+        capabilities.setCapability(FirefoxDriver.MARIONETTE, false);
+        capabilities.setCapability(FirefoxDriver.PROFILE, fireFoxProfile);
```
Comment (jpeerz) [inline orcid-integration-test/src/test/java/org/orcid/integration/blackbox/web/shibboleth/ShibbolethTest.java]: @wjrsimpson Is this line duplicate with 82 ?

File: TESTAUTO.md (hunk 1 above)
Comment (wjrsimpson) [inline TESTAUTO.md]: I've now suggested ~/bin for installation, but am not pre-populating the full path because we don't know the users home directory.

**Pull request merged**